### ISSUE TITLE
SEP-24 more info amount labels

### ISSUE THREAD

**Issue by yuriescl** [stellar/django-polaris#454]
`Amount To Be Received` and `Amount Deposited` can sound confusing to the user:

![image](https://user-images.githubusercontent.com/26092447/128122604-9dcf9412-8fb4-47a8-bb62-2724e388bcea.png)

User can interpret "amount deposited" as the amount to be paid (credit card, bank transfer), and "amount to be received" as the amount to be received in the stellar account.
This is not really an issue, just a small suggestion.

Maybe those labels can be changed to something like:
Amount you'll pay  (maybe "Amount you paid" once status changes)
Amount you'll receive in Stellar (maybe "Amount you received in Stellar" once it's completed)

**Comment by JakeUrban** [comment]
I like your suggestions because they're very direct and literal, but I'm trying to remove the "you" out of them as you don't see that language in UX's a lot.

Other money transfer services use language like the following:
<img width="392" alt="Screen Shot 2021-08-05 at 9 57 24 AM" src="https://user-images.githubusercontent.com/10968980/128390679-23e5a50f-b0a5-45d1-84fe-bcc0a75aee29.png">

I'm curious what you think of this. We could use this language or something similar?

**Comment by yuriescl** [comment]
The above language looks very good to me. I couldn't think of better labels that work for both deposits and withdrawals.
Once the user sees the fees are being added and he'll receive `$100`, then it's easy to understand what's happening.

**Comment by JakeUrban** [comment]
Is it ok if I mark this for the 1.6 release? I'd like to get input from other anchors using Polaris before changing their UX.

@ronaldoy 
@markusait
@brunopedrazza 
@pitsevich

**Comment by ronaldoy** [comment]
Yes, it is ok.

**Comment by brunopedrazza** [comment]
Sure.

> Is it ok if I mark this for the 1.6 release? I'd like to get input from other anchors using Polaris before changing their UX.
> 
> @ronaldoy
> @markusait
> @brunopedrazza
> @pitsevich

**Comment by antb123** [comment]
> The above language (Cost Summary) looks very good to me. I couldn't think of better labels that work for both deposits and withdrawals.
> Once the user sees the fees are being added and he'll receive `$100`, then it's easy to understand what's happening.

Hi Jake... not sure about calling it total cost. Maybe Total to Pay?

Here is an example

![image](https://user-images.githubusercontent.com/17188302/128408726-4c2cfa22-c95b-4be7-8bbb-ceab751b93c5.png)

**Comment by JakeUrban** [comment]
@antb123 "Total to Pay" / "Total Paid" works for me 👍

**Comment by JakeUrban** [comment]
I'm implementing this now, and I've realized that the examples @antb123 and I have provided are for remittances between two people, whereas the SEP-24 UI shows a transaction involving a single person.

This means there is no reason to show both "Send/Source Amount" and "Total Paid", since they'll always be the same.

If the user wants to receive 100USDC on Stellar, he'll have to send $103 to the anchor's bank account if the fee is $3:
>> Send Amount: $103
Fees: $3
Receive Amount: $100

It works the same for withdrawals. If the user wants to receive $100 in his bank, he'll have to send 103USDC to the anchor's distribution account:
>> Send Amount: $103
Fees: $3
Receive Amount: $100

There is no reason for "Total Paid" or "Total to Pay".
